src/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -404,7 +404,7 @@ std::vector<std::pair<ngraph::element::Type, std::vector<std::uint8_t>>> LayerTe
     auto refInputsTypes = std::vector<ngraph::element::Type>(inputs.size());
     for (std::size_t i = 0; i < inputs.size(); ++i) {
         const auto &input = inputs[i];
-        const auto &inputSize = input->byteSize();
+        const auto inputSize = input->byteSize();
```

Comment:
why this is modified?

Reply:
i was confused by binding of prvalue integer to reference.
probably it's just copy-paste of line above.